config/config.go

```diff
@@ -318,6 +320,7 @@ func NewConfig() *Config {
 	flag.Var(cfg.PredicatePlugins, "predicate-plugin", "set a custom predicate plugins to load, a comma separated list of name and arguments")
 	flag.Var(cfg.DataclientPlugins, "dataclient-plugin", "set a custom dataclient plugins to load, a comma separated list of name and arguments")
 	flag.Var(cfg.MultiPlugins, "multi-plugin", "set a custom multitype plugins to load, a comma separated list of name and arguments")
+	flag.Var(cfg.CompressEncodings, "compress-encodings", "set encodings supported for compression, when client has no preference, uses in priority in the order of this array")
```

Comment:
@AlexanderYastrebov is the description clear on the expected behaviour?

Reply:
Maybe something like
`set encodings supported for compression, the order defines priority when Accept-Header has equal quality values, see RFC 7231 section 5.3.1`